src/core/socket.c

```diff
@@ -72,6 +72,7 @@ static const UnitActiveState state_translation_table[_SOCKET_STATE_MAX] = {
 
 static int socket_dispatch_io(sd_event_source *source, int fd, uint32_t revents, void *userdata);
 static int socket_dispatch_timer(sd_event_source *source, usec_t usec, void *userdata);
+static void flush_ports(Socket *s);
```

Comment:
Please also rename this to match the option name if that makes sense after the option is renamed. (I mean e.g. flush_pending_connections() would be nice function name. But if the option name ends up being just Flush=, then function name flush() would be very cryptic.)

Reply:
`flush_ports()` was already implemented, I just reused it here. Do you propose to rename `flush_ports()` into `flush_pending_connections()`?

Reply:
In fact it would be more `flush_pending()` since not only incoming connections are flushed, but unread data also.

Reply:
I think flush_ports() is OK as it is. After all that function iterates through all SocketPort objects and flushes them, so the name is pretty appropriate I'd argue.

Reply:
> I think flush_ports() is OK as it is. After all that function iterates through all SocketPort objects and flushes them, so the name is pretty appropriate I'd argue.

@keszybz For now I renamed `flush_ports()` into `flush_pending()` but @poettering seems to prefer old name. Please advise, I'll then fix and squash the commits.

Reply:
FlushPending= is nice. As to the internal function name: flush_ports() is OK too.